### PR TITLE
Fixing squid:S2583: Conditions should not unconditionally evaluate to "TRUE" or to "FALSE"

### PR DESCRIPTION
**Pull request by asamaras11**
This pull request is focused on resolving occurrences of Sonar rule squid:S2583: Conditions should not unconditionally evaluate to "TRUE" or to "FALSE"
You can find more information about the issue here:
https://dev.eclipse.org/sonar/coding_rules#q=squid:S2583
Please let me know if you have any questions.
Aris Samaras

### CODE REVIEW

File: src/main/java/org/red5/server/net/proxy/DebugProxyHandler.java [hunk 1]
```diff
@@ -115,20 +115,18 @@ public void sessionCreated(IoSession session) throws Exception {
...
+        File rawFile = loader.getResource(dumpTo + fileName + ".raw").getFile();
+        rawFile.createNewFile();
 
-            File rawFile = loader.getResource(dumpTo + fileName + ".raw").getFile();
-            rawFile.createNewFile();
-
-            FileOutputStream headersFos = new FileOutputStream(headersFile);
+        FileOutputStream headersFos = new FileOutputStream(headersFile);
+        FileOutputStream rawFos = new FileOutputStream(rawFile);
+        try {
```
Comment (solomax) [inline src/main/java/org/red5/server/net/proxy/DebugProxyHandler.java]: I would use 
try (FileOutputStream headersFos = new FileOutputStream(headersFile); FileOutputStream rawFos = new FileOutputStream(rawFile);) {
...........
}
instead

Comment (asamaras11) [inline src/main/java/org/red5/server/net/proxy/DebugProxyHandler.java]: The compiler version at pom.xml is set at Java 1.6, where the try-with-resources command is not available. Do you want me to set it at 1.7?

Comment (solomax) [inline src/main/java/org/red5/server/net/proxy/DebugProxyHandler.java]: Actually red5 is being built using 1.7 and Paul was going to change it to 1.8 .....
1.6 was set for some compatibility reasons ....

Comment (mondain) [inline src/main/java/org/red5/server/net/proxy/DebugProxyHandler.java]: I bumped the pom's up-to 1.7, but they aren't committed yet; I don't want to push 1.8 just yet even though that's what everyone should be on by this point based on the JDK end-of-line dates.

**Pull request merged**